examples/clipboard/Clipboard.cpp

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
Do we really need an example for such a simple thing? Examples are rather demonstrating the use of one or more whole modules, or complex classes.